src/components/ResourceBrowser/ResourceList/ResourceList.component.tsx

```diff
@@ -87,3 +92,33 @@ export const ClusterOptionWithIcon = (props) => {
         </components.Option>
     )
 }
+
+export const KindSearchValueContainer = (props) => {
```

Comment:
Can we extract these in utils component

Reply:
It's a component so kept it in this file.